frontend/src/app/core/state/in-app-notifications/in-app-notifications.service.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  Injectable,
+  Injector,
+} from '@angular/core';
+import {
+  catchError,
+  tap,
+} from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import {
+  applyTransaction,
+  ID,
+} from '@datorama/akita';
+import { APIV3Service } from 'core-app/core/apiv3/api-v3.service';
+import { NotificationsService } from 'core-app/shared/components/notifications/notifications.service';
+import { InAppNotificationsStore } from './in-app-notifications.store';
+import { InAppNotification } from './in-app-notification.model';
+import { IHALCollection } from 'core-app/core/apiv3/types/hal-collection.type';
+import { HttpClient } from '@angular/common/http';
+import { InAppNotificationsQuery } from 'core-app/core/state/in-app-notifications/in-app-notifications.query';
+import { Apiv3ListParameters } from 'core-app/core/apiv3/paths/apiv3-list-resource.interface';
+import { collectionKey } from 'core-app/core/state/collection-store.type';
+import {
+  markNotificationsAsRead,
+  notificationsMarkedRead,
+} from 'core-app/core/state/in-app-notifications/in-app-notifications.actions';
+import {
+  EffectCallback,
+  EffectHandler,
+} from 'core-app/core/state/effects/effect-handler.decorator';
+import { UntilDestroyedMixin } from 'core-app/shared/helpers/angular/until-destroyed.mixin';
+import { ActionsService } from 'core-app/core/state/actions/actions.service';
+
+@EffectHandler
+@Injectable()
+export class InAppNotificationsService extends UntilDestroyedMixin {
```

Comment:
Perhaps we can differentiate UI stores/services from Resource stores services in name. 

e.g. `InAppNotificationResourceService` or smth. A bit mouthy, but otherwise it'll be hard to differentiate between them when reading dependent code